### ISSUE TITLE
Error: Unable to resolve module classnames 

### ISSUE THREAD

**Issue by student1060632fpt** [ant-design/ant-design-mobile-rn#1229]
### Reproduction link

[https://files.gitter.im/58ecaa31d73408ce4f57939e/kd2R/image.png](https://files.gitter.im/58ecaa31d73408ce4f57939e/kd2R/image.png)

### Steps to reproduce

1. Install antd for react native in mobile like in https://rn.mobile.ant.design/docs/react/introduce
2. Use Checkbox of @ant-design/react-native
```
const HomeScreen:FC = ({ navigation }: any) => {
  const [checked, setChecked] = useState(false)
  return (
    <View style={styles.container}>
      <Checkbox
       onChange={event => {
        setChecked(event.target.checked );
       }}
        checked={checked}
      />
    </View>
  );
};
const styles = StyleSheet.create({
  container: {
  },
});

export default HomeScreen;
```
3. Restart yarn and run react-native run-android to run the app in emulator
4. The node window display module
``` 
error: Error: Unable to resolve module classnames from C:\Users\hoa.nguyenlexuan\Desktop\vaultbox\Vaultbox-dev-Mobile\vaultbox_mobile\node_modules\@ant-design\react-native\lib\tag\index.js: classnames could not be found within the project or in these directories:
  node_modules\@ant-design\react-native\node_modules
  node_modules
```
Because the node window display Error so it makes the app display the 500 error too:
![This is an image](https://files.gitter.im/58ecaa31d73408ce4f57939e/kd2R/image.png)

### What is expected?

We should not see this error and it should display a checkbox

### What is actually happening?

The node window display module
``` 
error: Error: Unable to resolve module classnames from C:\Users\hoa.nguyenlexuan\Desktop\vaultbox\Vaultbox-dev-Mobile\vaultbox_mobile\node_modules\@ant-design\react-native\lib\tag\index.js: classnames could not be found within the project or in these directories:
  node_modules\@ant-design\react-native\node_modules
  node_modules
```
Because the node window display Error so it makes the app display the 500 error too:
![This is an image](https://files.gitter.im/58ecaa31d73408ce4f57939e/kd2R/image.png)

| Environment | Info |
| --- | --- |
| antd | 5.0.0 |
| React | React: 17.0.2 /react native: 0.68.1  |
| System | Android 10.0.0 (google api) |
| Browser | Android emulator - Pixel 5  |

**Comment by 1uokun** [comment]
```bash
npm install classnames rc-util
```
Duplicate of #1228

**Comment by student1060632fpt** [comment]
> ```shell
> npm install classnames rc-util
> ```
> 
> Duplicate of #1228

Thank a lot @1uokun, It's really work for me

**Comment by 1uokun** [comment]
fixed in `5.0.1`
